flaskutils/models.py

```diff
@@ -1,8 +0,0 @@
-from pgsqlutils.orm import BaseModel
-
-
-class FlaskModel(BaseModel):
```

Comment:
I suppose this means that our other repos may need to change their imports, if they do something like this : `from flaskutils.models import FlaskModel`?